### ISSUE TITLE
[IMPROVEMENT] - sort-orders.tpl and per-page.tpl it is necessary to add aria-label to their select tag

### ISSUE THREAD

**Issue by maofree** [Oksydan/falcon#350]
### Description

Hi
I report here how I've fixed the accessibility error on these two files
sort-orders.tpl and per-page.tpl

`<select data-action="search-select" class="custom-select" aria-label="{l s='Per page:' d='Shop.Theme.Catalog'}">`
`<select data-action="search-select" class="custom-select" aria-label="{l s='Sort order' d='Shop.Istheme'}">`

bye

### Node.js version

v16

### php version

8.1

### OS and it's version

linux

### Browsers

Chrome

### Required module/theme

theme

### Reproduction steps

```bash
1. look at sort-orders.tpl and per-page.tpl
```

**Comment by Oksydan** [comment]
Hi @maofree,

thank you for creating an issue.
It's should be more like feature request instead of bug report. 
There is nothing broken tbh it's just missing aria-label. 

I will add this to the version 4 roadmap.

**Comment by Oksydan** [comment]
Hi @maofree,

problem has been fixed on develop branch. 
We can now close this issue. Thank you for your contribution.
